### DISCUSSION TITLE
CPU clock slider

### DISCUSSION THREAD

**Discussion by GavanGX** [seerge/g-helper#2343] [Ideas]
Hey i just want to ask is it possible to add cpu clock slider on next update? I own asus tuf gaming and unfortunately my bios wont allow me to adjust the TDP nor undervolting features. My only option to reduce the temp is by disabling CPU boost. However disabling CPU boost limit my performance in some games so i need a way to limit the boost the certain clock speed. I'm aware you can do this by editing some files in regedit(which im not fond of). I'd preferred if i can do this in your apps. If possible please add this feature in future update. Thanks man.

**Comment by Kikimahe** [comment]
Hi,

You can already limit the CPU clock frequency in Windows Power Plan. #1907 ;-)

Use the [Power settings explorer](https://forums.guru3d.com/threads/windows-power-plan-settings-explorer-utility.416058/) software to unlock the “Max frequency Cpu” and “Max frequency CPU class 1” settings.

**Comment by GavanGX** [comment]
Hi im well aware of this however this required me to edit some registry
which im not comfortable with.

**Comment by seerge** [comment]
@GavanGX hello, as @Kikimahe has mentioned [it's possible to set that via power plan](https://github.com/seerge/g-helper/discussions/1907). I don't plant to add it to the app, as that option works on some CPUs and simply don't on others, so I don't want to get into this at all :) 

Thanks

**Comment by manciuszz** [comment]
I recommend "Quick CPU" for this - https://www.coderbag.com/product/quickcpu
![image](https://github.com/seerge/g-helper/assets/2539661/6a562a51-eae0-4b07-bdfb-65be416a4207)

It also has a built-in, easy to use Power Plan Manager.
![image](https://github.com/seerge/g-helper/assets/2539661/3d19213b-29a7-4cee-9d92-09bf8ffc0570)

**Comment by GavanGX** [comment]
Too bad for me. Anyway thanks for letting me know. Your app is great. I
hope to see more features introduced in the future. Keep up the good work.

**Reply by seerge** [reply]
You can unlock this setting in power plan it with powershell command

```
powercfg -attributes SUB_PROCESSOR 75b0ae3f-bce0-45a7-8c89-c9611c25e100 -ATTRIB_HIDE
```

**Reply by Anres1981** [reply]
`
powercfg /SETDCVALUEINDEX SCHEME_CURRENT SUB_PROCESSOR PROCFREQMAX 3500

powercfg /SETACVALUEINDEX SCHEME_CURRENT SUB_PROCESSOR PROCFREQMAX 3500
`
